modules/services/servers/src/main/java/au/com/shiftyjelly/pocketcasts/servers/model/DiscoverModel.kt

```diff
@@ -64,6 +64,7 @@ data class DiscoverRow(
     @field:Json(name = "sponsored") val sponsored: Boolean = false,
     @field:Json(name = "curated") override val curated: Boolean = false,
     @field:Json(name = "sponsored_podcasts") val sponsoredPodcasts: List<SponsoredPodcast> = emptyList(),
+    val regionCode: String? = null,
```

Comment:
Is this part of a JSON object? If it is we should annotate it with `@Json` like all other properties. If it is not I don't think we should be adding it here as it mixes layers.